### PR TITLE
[IOS-649]: Enable network switching on all targets, show mainnet and ropsten for production

### PR DESCRIPTION
**Pull request by yuliaveres**

### CODE REVIEW

File: Toshi/Network/NetworkSwitcher.swift [hunk 1]
```diff
@@ -106,7 +106,7 @@ final class NetworkSwitcher {
         #if DEBUG || TOSHIDEV
             return [.ropstenTestNetwork, .toshiTestNetwork]
         #else
-            return [.ropstenTestNetwork]
+            return [.mainNet, .ropstenTestNetwork]
```
Comment (designatednerd) [inline Toshi/Network/NetworkSwitcher.swift]: a) do we also want to allow the toshi test network here? 
b) Is there anything that needs to change in terms of what's happening with the `TokenURLPaths` in addition to these changes to make this work?

Comment (yuliaveres) [reply Toshi/Network/NetworkSwitcher.swift]: a) as you see - no
b) again - no until we have additional test networks, which will mean - just few new URLs

Comment (designatednerd) [reply Toshi/Network/NetworkSwitcher.swift]: Right I saw that it's not there, I was wondering if that was an intentional omission

**Comment by designatednerd** [comment]
I'm gonna merge this since Yulia's down in 🇦🇺 at the moment and done with her day.

**Comment by yuliaveres** [comment]
Thanks @designatednerd

**Pull request merged**